### PR TITLE
FIT-475 - Create Master Pipeline for Component Library

### PR DESCRIPTION
**Pull request by SergiiRedchuk**
timestamps test

### CODE REVIEW

**Comment by SergiiRedchuk** [comment]
retest this please

**Comment by SergiiRedchuk** [comment]
retest this please

File: jenkinsfile.groovy [hunk 1]
```diff
@@ -57,6 +58,30 @@ def pullRequestPipeline() {
...
+      currentBuild.result = 'FAILURE'
+      throw exception
+    } finally {
+      cleanupStage()
+    }
+  }
+}
+
+/**
+ * Creates pull request pipeline.
```
Comment (eckdanny-osi) [inline jenkinsfile.groovy]: this GroovyDoc is not correct

Comment (SergiiRedchuk) [reply jenkinsfile.groovy]: fixed

File: jenkinsfile.groovy [hunk 2]
```diff
@@ -57,6 +58,30 @@ def pullRequestPipeline() {
           sh "yarn build:www"
         }
       }
+    } catch (Exception exception) {
```
Comment (eckdanny-osi) [inline jenkinsfile.groovy]: I'm confused why this is showing up in the diff. This block is in `master` here:  https://github.com/ca-cwds/design-system/blob/11b80ce7257af1cc7d10ad153e35fb72e8b3ebd0/jenkinsfile.groovy#L60-L66

Do we need a merge from `master` into this?

File: jenkinsfile.groovy [hunk 3]
```diff
@@ -8,15 +8,17 @@ def GITHUB_CREDENTIALS_ID = '433ac100-b3c2-4519-b4d6-207c029a103b'
...
-switch (env.BUILD_JOB_TYPE) {
-  case 'master':
-    // masterPipeline()
-    break
-  case 'release':
-    // releasePipeline()
-    break
-  default:
-    pullRequestPipeline()
+timestamps {
```
Comment (eckdanny-osi) [inline jenkinsfile.groovy]: 👍

File: jenkinsfile.groovy (hunk 2 above)
Comment (SergiiRedchuk) [reply jenkinsfile.groovy]: diff because of a space after 'catch'

Comment (SergiiRedchuk) [reply jenkinsfile.groovy]: Sorry, don't understand the question: "Do we need a merge from master into this?"

**Pull request merged**